remove Python2 support in command_line_runner

### Description 
README.md says,
> Unfortunately Python 2.7 support has been discontinued :(

I think we can drop support for the older version of Python [here](https://github.com/gleitz/howdoi/blob/f19ca79c7aff69e09c5393b2ede6ab24cddf6f75/howdoi/howdoi.py#L816).

I am not sure if we still need utf-8 encoding.
Maybe we can do like this?

```
utf8_result = howdoi(args).encode('utf-8', 'ignore')
sys.stdout.buffer.write(utf8_result)

# or
result = howdoi(args)
print(result)
```

### Have you read the Contributing Guidelines on Pull Requests on mkdocs?

Yes

I forked master and clone into local. Then I ran tests without any change to code. I get one failure but have no idea whats causing it. Can someone help me out?

```
======================================================================
FAIL: test_missing_pre_or_code_query (__main__.HowdoiTestCase)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/XXX/Projects/howdoi/test_howdoi.py", line 270, in test_missing_pre_or_code_query
    self.assertIn('XML elements present in a XML', output)
AssertionError: 'XML elements present in a XML' not found in '<people>\n    <student>\n        <name>John</name>\n    </student>\n</people>\n'

----------------------------------------------------------------------
```

--- a/howdoi/howdoi.py
+++ b/howdoi/howdoi.py
@@ -812,12 +812,9 @@ def command_line_runner():  # pylint: disable=too-many-return-statements,too-man
     if os.getenv('HOWDOI_COLORIZE'):
         args['color'] = True
 
-    utf8_result = howdoi(args).encode('utf-8', 'ignore')
-    if sys.version < '3':
-        print(utf8_result)
-    else:
-        # Write UTF-8 to stdout: https://stackoverflow.com/a/3603160
-        sys.stdout.buffer.write(utf8_result)
+    result = howdoi(args)
+    print(result)
+
     # close the session to release connection
     howdoi_session.close()
 

--- a/howdoi/howdoi.py
+++ b/howdoi/howdoi.py
@@ -812,8 +812,9 @@ def command_line_runner():  # pylint: disable=too-many-return-statements,too-man
     if os.getenv('HOWDOI_COLORIZE'):
         args['color'] = True
 
-    result = howdoi(args)
-    print(result)
+    utf8_result = howdoi(args).encode('utf-8', 'ignore')
+    # Write UTF-8 to stdout: https://stackoverflow.com/a/3603160
+    sys.stdout.buffer.write(utf8_result)
 
     # close the session to release connection
     howdoi_session.close()

